app/containers/ErrorBoundary/index.js

```diff
@@ -45,22 +45,24 @@ export class ErrorBoundary extends React.Component {
       translations,
     };
 
-    return [
-      sendProps.error || sendProps.errorInfo ? (
-        <div key="errorBoundary">
-          <Helmet>
-            <title>{translations[messages.title.id]}</title>
-            <meta
-              name="description"
-              content={translations[messages.description.id]}
-            />
-          </Helmet>
-          <ErrorBoundryMessage {...sendProps} />
-        </div>
-      ) : (
-        React.Children.only(this.props.children)
-      ),
-    ];
+    return (
+      <React.Fragment>
+        {sendProps.error || sendProps.errorInfo ? (
+          <div>
```

Comment:
Is this `<div>` needed? It is already wrapped in `<React.Fragement>`

Reply:
You are right. I place it wrong here

Reply:
Yes, I use <div> if there is necessity to place ErrorMessage. And opposite - use React.Fragment if I need to insert childrens. I ll refactor this component a bit.